src/material/stepper/stepper.md

```diff
@@ -222,3 +222,19 @@ is given `role="tab"`, and the content that can be expanded upon selection is gi
 step content is automatically set based on step selection change.
 
 The stepper and each step should be given a meaningful label via `aria-label` or `aria-labelledby`.
+
+When a stepper is used on a smaller screen, verical steppers are preferred over
+horizontal steppers because the text in the header will be more visible.
```

Comment:
Might be worth mentioning here that there's an example of a responsive stepper further up in the doc.